modules/signals/entities/src/helpers.ts

```diff
@@ -7,6 +7,7 @@ import {
   SelectEntityId,
 } from './models';
 
+declare const ngDevMode: unknown;
```

Comment:
1) There is an option to avoid declaring `ngDevMode` in each file where it's needed by using:

```ts
declare global {
  declare const ngDevMode: unknown;
}
```

in a single place. However, in that case, the `ngDevMode` global variable will be available in applications that use the `@ngrx/signals` package.

2) I also tried to define `ngDevMode` in a separate file and export it as renamed `ngrxDevMode`:

```ts
// file: dev-mode.ts
declare const ngDevMode: unknown;
export const ngrxDevMode = ngDevMode;

// file: helpers.ts
import { ngrxDevMode } from './dev-mode';


if (ngrxDevMode) {
  // ...
}
```

However, tree-shaking will not work in apps if `ngDevMode` is renamed. cc @brandonroberts

Reply:
Good catch. We'll go with it as-is for now and adjust if they change it in the framework 👍